scripts/make-pot.mjs

```diff
@@ -34,17 +34,17 @@ const commands = [
 		description: 'Generating pot file with wp-babel-makepot',
 	},
 	{
-		command: `open "${ IMPORT_PAGE }"`,
-		description: `Opening the translation import page ${ IMPORT_PAGE } in the browser`,
+		command: `open -R "${ POT_FILE }"`,
+		description: `Revealing pot file in Finder: ${ POT_FILE }`,
 	},
 	{
 		command: 'sleep 0.5',
-		description: 'Waiting for the browser to open',
+		description: 'Waiting for the Finder to open',
 	},
```

Comment:
Let's remove the sleep step as I think it's not necessary anymore. I originally added it to "force" the finder to open on top of the browser.